jenkins-jobs/jobs/ShuffleboardJob.groovy

```diff
@@ -84,7 +84,7 @@ def setupBuildSteps(job, usePublish, properties = null) {
         if (usePublish) {
             publishers {
                 archiveArtifacts {
-                    pattern('app/build/libs/app-all.jar')
+                    pattern('app/build/libs/*.jar')
```

Comment:
We also need to publish the API and plugins. Not just the app uberjars

Reply:
@333fred What happens if multiple files match the star?

Reply:
It'll archive all matching artifacts.